pre-check/v1.1.x/check.sh

```diff
@@ -176,12 +176,20 @@ check_volumes()
                 echo checking running engine "${lh_engine}..."
 
                 if [ $node_count -gt 2 ];then
-                    robustness=$(kubectl get volumes.longhorn.io/$lh_volume -n longhorn-system -o jsonpath='{.status.robustness}')
-                    if [ "$robustness" = "healthy" ]; then
-                        echo "Volume $lh_volume is healthy."
-                    else
-                        echo "Volume $lh_volume is degraded."
+                    volume_json=$(kubectl get volumes.longhorn.io/$lh_volume -n longhorn-system -o json)
+                    # single-replica volumes should be handled exclusively
+                    volume_replicas=$(echo $volume_json | jq -r '.spec.numberOfReplicas')
+                    if [ $volume_replicas -eq 1 ]; then
+                        echo "Volume $lh_volume is a single-replica volume, please consider shut down the corresponding workload before upgrading."
```

Comment:
```suggestion
                        echo "Volume $lh_volume is a single-replica volume, please consider shut down the corresponding workload or adjusting its replica count before upgrading."
```

Just provide another option.

Reply:
Updated, thanks!